plugins/inputs/win_eventlog/win_eventlog.go

```diff
@@ -426,7 +426,7 @@ func (w *WinEventLog) renderEvent(eventHandle EvtHandle) (Event, error) {
 func (w *WinEventLog) renderLocalMessage(event Event, eventHandle EvtHandle) (Event, error) {
 	publisherHandle, err := openPublisherMetadata(0, event.Source.Name, w.Locale)
 	if err != nil {
-		return event, nil
+		return event, nil //nolint:nilerr // We can return event without most values
```

Comment:
I wrote the code, but I cannot remember why we return nil in case of error. Most of the event fields will be useless in that case, so I think we should actually return the error here... @powersj what do you think?

Reply:
@srebhan almost the same behaviour can be observed in lines 407-412. Maybe not fully completed events are ok for these cases?

Reply:
> Maybe not fully completed events are ok for these cases?

agreed - I vaguely recall @srebhan working through this issue related to events not being complete or missing data after a reboot and we said we would ignore those

Reply:
Then this is ok for me. Thanks guys!